cli/dcoscli/tables.py

```diff
@@ -639,14 +639,37 @@ def queued_app_details_table(queued_app):
     :rtype: PrettyTable
     """
 
+    def value_declined(entry, value):
+        """Returns `yes` if the value was inside entry.get('reason'),
+        returns `no` otherwise.
+
+        :param entry: row entry
+        :type entry: [dict]
+        :param value: value which should be checked
+        :type value: string
+        :rtype: PrettyTable
+        """
+        if value not in entry.get('reason', []):
+            """check sumbol"""
+            return u'\u2713'
+        else:
+            """cross symbol"""
+            return u'\u2717'
+
     reasons = queued_app.get('lastUnusedOffers')
     fields = OrderedDict([
         ('HOSTNAME', lambda entry:
             entry.get('offer', {}).get('hostname', EMPTY_ENTRY)
          ),
-        ('REASON', lambda entry:
-            ', '.join(entry.get('reason'))
+        ('ROLE', lambda entry: value_declined(entry, 'UnfulfilledRole')),
+        ('CONSTRAINTS', lambda entry:
+            value_declined(entry, 'UnfulfilledConstraint')
          ),
+        ('CPUS', lambda entry: value_declined(entry, 'InsufficientCpus')),
+        ('MEM', lambda entry: value_declined(entry, 'InsufficientMemory')),
+        ('DISK', lambda entry: value_declined(entry, 'InsufficientDisk')),
+        ('GPUS', lambda entry: value_declined(entry, 'InsufficientGpus')),
```

Comment:
Until we have a general way for users to enable/disable columns, I'd rather just have it turned off for now. The work to get preferences saved will be happening in 1.10